frontend/src/views/Combat.vue

```diff
@@ -69,6 +69,13 @@
                 <b-button v-if="selectedWeaponId" variant="primary" class="ml-3" @click="selectedWeaponId = null" id="gtag-link-others" tagname="choose_weapon">
                   Choose New Weapon
                 </b-button>
+
+                <b-button v-if="selectedWeaponId" variant="primary" class="ml-3 smoke-bomb-button"
+                @click="useSmokeBombUI()" id="gtag-link-others" tagname="use_smokebomb"
+                :disabled="smokeBombs == 0">
```

Comment:
:disabled="smokeBombs ===  0"> 
Hmm, linter should suggest this change, do you have it set up correctly in your IDE? Also, as 0 is falsy, it can be just :disabled="!smokeBombs" (if smokeBombs is 0 it will be disabled, otherwise it will be enabled)

Reply:
for some reason it's only working as ==

Reply:
It works when === and string ('0'); thanks for the tip